### PR TITLE
Add unit test for embulk-input-ftp sub project

### PR DESCRIPTION
**Pull request by sakama**
* Add unit test for embulk-input-ftp sub project
* Added description about how to prepare/execute test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,3 +110,56 @@ $ ./gradlew bintrayUpload # release embulk-input-sftp to Bintray maven repo
...
+FTP_TEST_USER
+FTP_TEST_PASSWORD
+FTP_TEST_SSL_TRUSTED_CA_CERT_FILE
+FTP_TEST_SSL_TRUSTED_CA_CERT_DATA
+```
+
+Following option is optional
+```
+FTP_TEST_PORT (default:21)
+FTP_TEST_SSL__PORT (default:990)
```
Comment (kamatama41) [inline README.md]: Is `__` (two underscores) a typo?

File: README.md [hunk 2]
```diff
@@ -110,3 +110,56 @@ $ ./gradlew bintrayUpload # release embulk-input-sftp to Bintray maven repo
...
+```
+
+If you're using Mac OS X El Capitan and GUI Applications(IDE), like as follows.
+```xml
+
+launchctl setenv FTP_TEST_SSL_TRUSTED_CA_CERT_FILE$ vi ~/Library/LaunchAgents/environment.plist
```
Comment (kamatama41) [inline README.md]: The `launchctl setenv ..` here is necessary?

File: embulk-input-ftp/src/test/java/org/embulk/input/TestFtpFileInputPlugin.java [hunk 3]
```diff
@@ -1,5 +1,314 @@
...
+
+    /*
+     * This test case requires environment variables
+     *   FTP_TEST_HOST
+     *   FTP_TEST_USER
+     *   FTP_TEST_PASSWORD
+     *   FTP_TEST_SSL_TRUSTED_CA_CERT_FILE
+     */
+    @BeforeClass
+    public static void initializeConstant()
```
Comment (kamatama41) [inline embulk-input-ftp/src/test/java/org/embulk/input/TestFtpFileInputPlugin.java:59]: `FTP_TEST_SSL_PORT`, `FTP_TEST_SSL_TRUSTED_CA_CERT_FILE` and `FTP_TEST_DIRECTORY` seem not to be used. Are these variables going to be used future?

**Review by kamatama41 — COMMENTED** [review]
Basically LGTM 👍 I put some minor questions.

File: README.md (hunk 1 above)
Comment (sakama) [reply README.md]: Thanks. Fixed at 6de6352

File: README.md (hunk 2 above)
Comment (sakama) [reply README.md]: Fixed at 6de6352

File: embulk-input-ftp/src/test/java/org/embulk/input/TestFtpFileInputPlugin.java (hunk 3 above)
Comment (sakama) [reply embulk-input-ftp/src/test/java/org/embulk/input/TestFtpFileInputPlugin.java:59]: > Are these variables going to be used future?
Yeah and I often use them when I develop at my local env.
But other developers usually don't have FTPS environments.

**Pull request merged**